### PR TITLE
Added visible representation of line endings for diffs

### PR DESCRIPTION
**Pull request by mcustiel**
This tries to be a fix for https://github.com/sebastianbergmann/phpunit/issues/2148

### CODE REVIEW

**Comment by mcustiel** [comment]
If you consider this a good approach, I will add a commit with the fixed tests.

File: src/Exporter.php [hunk 1]
```diff
@@ -236,7 +236,7 @@ protected function recursiveExport(&$value, $indentation, $processed = null)
             }
 
             return "'" .
-            str_replace(array("\r\n", "\n\r", "\r"), array("\n", "\n", "\n"), $value) .
+            str_replace(array("\r\n", "\n\r", "\r", "\n"), array('<crlf>', '<lfcr>', '<lf>', '<ln>'), $value) .
```
Comment (sebastianbergmann) [inline src/Exporter.php]: Why not print `\n`, for instance, when there is an `\`?

```
php > $s = "foo\n";
php > print $s;
foo
php > print str_replace("\n", "\\n\n", $s);
foo\n
```

Comment (mcustiel) [inline src/Exporter.php]: Also like the idea, Will do it like that.

Comment (sebastianbergmann) [inline src/Exporter.php]: You also need to make sure that this only gets applied to a string. Look at the test failures in https://travis-ci.org/sebastianbergmann/exporter/jobs/217512272 to see what I mean.

Comment (sebastianbergmann) [inline src/Exporter.php]: The Unicode `NEL` character (see https://en.wikipedia.org/wiki/Newline) might be useful here.

**Comment by mcustiel** [comment]
This is blocked by: https://github.com/sebastianbergmann/exporter/issues/20

**Comment by sebastianbergmann** [comment]
Am I correct to assume that since #20 was closed that this is no longer blocked?

**Comment by mcustiel** [comment]
Yes. I misunderstood the check for non-printable characters and it was not an issue. This is not blocked anymore.

**Pull request merged**